projects/aca-folder-rules/src/lib/services/folder-rules.service.ts

```diff
@@ -0,0 +1,79 @@
+/*!
+ * @license
+ * Alfresco Example Content Application
+ *
+ * Copyright (C) 2005 - 2020 Alfresco Software Limited
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Injectable } from '@angular/core';
+import { AlfrescoApiService } from '@alfresco/adf-core';
+import { BehaviorSubject, from, Observable } from 'rxjs';
+import { finalize, map } from 'rxjs/operators';
+import { Rule } from '../model/rule.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class FolderRulesService {
+  private rulesListingSource = new BehaviorSubject<Rule[]>([]);
+  rulesListing$: Observable<Rule[]> = this.rulesListingSource.asObservable();
+  private loadingSource = new BehaviorSubject<boolean>(false);
+  loading$ = this.loadingSource.asObservable();
+  constructor(private apiService: AlfrescoApiService) {}
+
+  loadRules(nodeId: string, ruleSetId: string = '-default-'): void {
```

Comment:
Shouldn't we name this method `getRules`?

Reply:
`loadRules` is appropriate in my opinion, since it doesn't return the rules itself but outputs them to the observable defined higher up in the service